tests/Dumplie/Spec/Application/Command/Inventory/CreateProductHandlerSpec.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Spec\Dumplie\Application\Command\Inventory;
+
+use Dumplie\Application\Command\Inventory\CreateProduct;
+use Dumplie\Application\Transaction\Factory as TransactionFactory;
+use Dumplie\Application\Transaction\Transaction;
+use Dumplie\Domain\Inventory\Product;
+use Dumplie\Domain\Inventory\Products;
+use Dumplie\Domain\SharedKernel\Money\Price;
+use Dumplie\Domain\SharedKernel\Product\SKU;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class CreateProductHandlerSpec extends ObjectBehavior
```

Comment:
Yeah when I will write some integration tests later we can think about removing this spec. It is basically how I works - create specification first so I not forgot about for example transaction commit ;)